src/test/resources/cleanup.sql

```diff
@@ -0,0 +1,4 @@
+USE discount_web_app;
```

Comment:
let's move this file to a separate directory
resources/sql/cleanup.sql for instance